metron-stellar/stellar-common/src/main/java/org/apache/metron/stellar/dsl/functions/MathFunctions.java

```diff
@@ -25,17 +25,39 @@
 import org.apache.metron.stellar.dsl.StellarFunction;
 
 import java.util.List;
+import java.util.function.Function;
 
 public class MathFunctions {
 
-  @Stellar(name="ABS"
-          ,description="Returns the absolute value of a number."
-          ,params = {
-                "number - The number to take the absolute value of"
-                    }
-          , returns="The absolute value of the number passed in."
-          )
-  public static class Abs implements StellarFunction {
+  private enum SingleArgMathFunctions implements Function<Double, Double> {
```

Comment:
I can certainly pull this out if you think it's useful.  I didn't think it was super useful outside of a helper enum for this class.